core/src/main/scala/org/apache/spark/deploy/security/HadoopFSDelegationTokenProvider.scala

```diff
@@ -99,11 +100,24 @@ private[deploy] class HadoopFSDelegationTokenProvider
   private def fetchDelegationTokens(
       renewer: String,
       filesystems: Set[FileSystem],
-      creds: Credentials): Credentials = {
+      creds: Credentials,
+      hadoopConf: Configuration,
+      sparkConf: SparkConf): Credentials = {
+
+    // The filesystems excluded from token renewal
+    val fsToExclude = sparkConf.get(KERBEROS_FILESYSTEM_RENEWAL_EXCLUDE)
+      .map(new Path(_).getFileSystem(hadoopConf).getUri.getHost)
+      .toSet
 
     filesystems.foreach { fs =>
-      logInfo(s"getting token for: $fs with renewer $renewer")
-      fs.addDelegationTokens(renewer, creds)
+      if (fsToExclude.contains(fs.getUri.getHost)) {
```

Comment:
Since this excludes all file systems on that host, shall we revise the description in line 107?

Reply:
`The hosts on which the file systems to be excluded from token renewal`?

Reply:
Ya. Maybe. 
BTW, we are accept `hadoopFileSystems` via 'spark.kerberos.renewal.exclude.hadoopFileSystem' and convert it to hosts?
Could you describe some examples where we cannot accept `hosts` directly?

Reply:
I'm also wondering why unlike the MR config we don't use hosts directly but rather URLs.

Reply:
No, we can accept hosts directly. It doesn't matter here. Either URL or host can achieve the same result we want.

It takes file system URLs because I let it to be consistent with other configs like `KERBEROS_FILESYSTEMS_TO_ACCESS`.